arbitrum-docs/bold/concepts/bold-technical-deep-dive.md

```diff
@@ -0,0 +1,234 @@
+---
+title: 'BoLD: a technical deep dive'
+sidebar_label: 'Technical deep dive'
+description: 'A technical deep dive into the BoLD protocol.'
+user_story: 'As a user or researcher of the Arbitrum product suite, I want to understand the technical choices behind the BoLD protocol.'
+content_type: concept
+author: leeederek
+target_audience: 'Developers, users and researchers interested in the Arbitrum BoLD protocol.'
+sme: leeederek
+sidebar_position: 1
+---
+
+import PublicPreviewBannerPartial from '../../how-arbitrum-works/bold/partials/_bold-public-preview-banner-partial.md';
+
+
+<PublicPreviewBannerPartial />
+
+## Overview
+
+Under the hood, BoLD can offer time-bounded, permissionless validation because a correct Arbitrum state assertion is not tied to a single validator or entity. Instead, claims are tied to deterministic Merkle proofs and hashes that will be proven on Ethereum. Any party can bond on the correct state and, through interactive fraud proofs, prove their claim is correct. This means that a single honest party bonding on the correct state assertion will always win disputes, guaranteed.
+
+To put it simply, Arbitrum’s current dispute protocol assumes that any assertion that gets challenged must be defended against by each unique challenger. It is similar to a 1-vs-1 tournament, where the honest party may participate in one or more concurrent tournaments at any time. BoLD, on the other hand, enables an all-vs-all battle royal between two categories: Good vs Evil, where there must and will always be a single winner in the end.
+
+Validators on Arbitrum can post their claim on the validity of state roots, known as **assertions**. Ethereum, of course, does not know anything about the validity of these Arbitrum state roots, but it _can_ help prove their correctness. _Anyone_ in the world can then initiate a challenge over any unconfirmed assertion to start the protocol’s game.
+
+The assertions being disputed concern block hashes of an Arbitrum chain at a given batch/inbox position. Given that Arbitrum chains are deterministic, there is only one correct history for all parties running the standard Nitro software. Using the notion of one-step proof, Ethereum can check whether someone is making a fraudulent assertion.
+
+If a claim is honest, it can be confirmed on Ethereum after a 6.4-day period (although the DAO can change this period). If a claim is malicious, anyone who knows the correct Arbitrum state can successfully challenge it within that 6.4-day window _and always win_ within a challenge period.
```

Comment:
We should define that 1 challenge period is 6.4d in length and that a challenge period is the time in which entities can open challenges!

Reply:
How would you like to include that definition?

Reply:
done.